PROJECT

```diff
@@ -81,4 +81,12 @@ resources:
   kind: DatadogDashboard
   path: github.com/DataDog/datadog-operator/api/datadoghq/v1alpha1
   version: v1alpha1
+- api:
+    crdVersion: v1
+    namespaced: true
+  domain: com
+  group: datadoghq
+  kind: DatadogPodAutoscaler
+  path: github.com/DataDog/datadog-operator/api/datadoghq/v2alpha1
```

Comment:
I don't think we'll expose a v2 for GA, so that version will probably v1beta1 or v1alpha2

Reply:
Done